src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -107,19 +130,10 @@ struct
       (Consensus.Protocol_state)
 
   module Transaction = struct
-    module T = struct
-      type t = Coda_base.Transaction.t =
-        | User_command of User_command.With_valid_signature.t
-        | Fee_transfer of Fee_transfer.t
-        | Coinbase of Coinbase.t
-      [@@deriving compare, eq]
-    end
-
-    include T
+    open Coda_base.Transaction
+    include Stable.Latest
 
-    include (
-      Coda_base.Transaction :
-        module type of Coda_base.Transaction with type t := t )
+    let fee_excess, supply_increase = (fee_excess, supply_increase)
```

Comment:
Does this statement have any effect? It looks to be shadowing the existing definitions.